tests/test_intervention.py

```diff
@@ -577,24 +577,15 @@ def test_card_html_update(
     test_user = db.session.merge(test_user)
     test_user.organizations.append(metastatic_org)
     consent = UserConsent.query.filter(
-        UserConsent.user_id == test_user.id).one()
+        UserConsent.user_id == test_user_id).one()
     consent.organization_id = metastatic_org.id
     consent.acceptance_date = dt
 
-    with SessionScope(db):
-        d = {'function': 'update_card_html_on_completion',
-             'kwargs': []}
-        strat = AccessStrategy(
-            name="update assessment_engine card_html on completion",
-            intervention_id=ae_id,
-            function_details=json.dumps(d))
-        db.session.add(strat)
-        db.session.commit()
-    user, ae, metastatic_org = map(
-        db.session.merge, (test_user, ae, metastatic_org))
+    # Fetch home page, which should include Assessment Engine "card"
+    results = client.get("/home")
 
     # without completing an assessment, card_html should include username
-    assert user.display_name in ae.display_for_user(user).card_html
+    assert bytes(test_user.display_name, 'utf-8') in results.data
```

Comment:
surprised `results.data` isn't a string...

Reply:
yup.  joys of py3, i couldn't find a simpler way to look for a "string" in the resulting HTML - open to suggestions :)